lib/ex.coffee

```diff
@@ -32,21 +32,19 @@ trySave = (func) ->
 
   deferred.promise
 
-saveAs = (filePath) ->
-  editor = atom.workspace.getActiveTextEditor()
-  fs.writeFileSync(filePath, editor.getText())
-
 getFullPath = (filePath) ->
-  if filePath is ''
-    throw new Error
+  editor = atom.workspace.getActiveTextEditor()
   if path.isAbsolute(filePath)
     return filePath
-  else if atom.workspace.getActiveTextEditor().getPath()?
-    return path.join(path.dirname(atom.workspace.getActiveTextEditor().getPath()), filePath)
-  else if atom.project.getPaths()[0]?
+  else if editor.getPath()?
+    if filePath is ''
+      return path.join(editor.getPath(), editor.getTitle())
+    else
+      return path.join(editor.getPath(), filePath)
+  else if atom.project.getPaths()[0]? and filePath isnt ''
```

Comment:
Although maybe it would make more sense to just use atom.workspace.getActiveTextEditor().buffer.getPath()